### DISCUSSION TITLE
Cannot save embedded file.

### DISCUSSION THREAD

**Discussion by sanchon68** [sumatrapdfreader/sumatrapdf#3034] [Q&A]
hello!
SumatraPDE v3.4.6 64-bit could not save the embedded file.
How can I solve it?

**Comment by GitHubRulesOK** [comment]
Sadly, best advice is use a different version for the task 
A portable copy of 3.1.2 is my secondary selection, for when some tasks are different to my primary pre-release install
you could use a portable 3.5.## pre-release as that also should allow extraction
